models/bonkbot/solana/bonkbot_solana_trades.sql

```diff
@@ -0,0 +1,152 @@
+{{ config(
+    alias = 'trades',
+    schema = 'bonkbot_solana',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['blockchain', 'tx_id', 'tx_index', 'outer_instruction_index', 'inner_instruction_index', 'token_bought_address', 'token_sold_address'],
+    post_hook='{{ expose_spells(\'["solana"]\',
```

Comment:
do you need the spell exposed on data explorer right away?
i wonder if we leave out for now, with idea of a larger bot spell to come that makes more sense to expose

Reply:
at this time, we don't have an automated process to remove from data explorer, so i try to be a bit careful upfront. it can always be added later, if you don't end up building a multi-project, crosschain bot spell that this feeds into. you can definitely still query it!